docs/properties/recommended_optional/property_distribution.rst

```diff
@@ -75,53 +90,53 @@ Examples:
 
 See :doc:`/guidance/distribution` for recommendations on archive file formats.
 
-.. _21.1.a:
+.. _21.1.1.a:
 
-21.1.a lastUpdated
-^^^^^^^^^^^^^^^^^^^^^
+21.1.1.a lastUpdated
+##########################
 
 **Occurrences:** 0-1
 
-**Definition:** Date when the content URL value was last updated. [#f1]_
+**Definition:** Date when the contents of the file were last updated.
```

Comment:
I think [in RFC ](https://github.com/datacite/schema-docs/issues/7#issuecomment-1235258151)people already gave a very good explanation about this. Checksum is best way to check for changes in the content of the file. As it  has been mentioned in several calls lastUpdated serves a different purpose. As defined It helps to identify path changes, file renaming, redirections and link roting; none of this is related to the content . Harvesters have only the metadata to make their judgement. It needs to be self-contained. 

I’m of the opinion that rather than evicerate `lastUpdated` of meaning, we removed it from this release and discuss it in future releases.

Reply:
Kristian - have you found users that actually care about path changes, file renaming, redirections, and link-roting (OK, there are users that care about link-roting)? LastUpdated is the date that the data last changed in every metadata standard I have ever worked on. We are not evicerating it, we are using it in a standard way. Also, it is not about checking for changes, it is about knowing when those changes happened.

Reply:
If the user wants to know when the value of the property last changed, this is already captured by our [activities](https://api.datacite.org/activities) for all properties, so it is redundant information. There isn't a reason for us to ask the user to provide it.

One scenario for harvesters is filtering for content that has been updated after X date. That will be made possible with the API using the revised definition of lastUpdated. Checksums would not be efficient here because they would have to store *all* previous checksums and compare with all current checksums to know what has been changed.

Reply:
I am with Kelly. The change of DataCite metadata is tracked, so there is no need to imply it in distribution. The aspect with the comparison of the checksums also seems plausible to me. So, in order not to inflate the scheme unnecessarily, it probably makes sense to omit lastUpdated for the current release.

Reply:
Yes, there are. You can find the use cases in the sub-group look in the [WG board - Top-right -](https://miro.com/app/board/uXjVOLwzj6w=/?share_link_id=14981602893). The challenge the sub-group was tackling was: `HMW deal updating the ContentUrl when a repository doesn't exist anymore or changes the path without updating the metadata?` Currently, DataCite has [500K DOIs](https://doi.datacite.org/providers/qjzc/dois) that have no administrator, or their administrator is inactive. This is due to either Member severing their membership with DataCite or going out of business. 
You can find all the solutions proposed in the document. In June, the WG sub-group decided that in the case  **contentUrl**, using **lastUpdated** sub-property would help metadata consumers looking at the document to discern of wrong values from out-of-date values for contentUrl when the response is a 404.


As I said probably is better not to include this sub-property as it is rather than include it with a different meaning.

Just to keep clear about metadata updates, metadata harvesters use the default **updated** to make delta harvesting. **updated** is about the metadata deposit. Our harvesters use that standard in our OAI and REST services.

Reply:
Decision to remove lastUpdated as it is redundant with the activities piece. If checksum is not enough, we can revisit at a later date.